### PR TITLE
Adjust bats-puppet test to look for hostname instead of the last host…

### PR DESCRIPTION
**Pull request by zjhuntin**
… in the list

### CODE REVIEW

File: bats/fb-test-puppet.bats [hunk 1]
```diff
@@ -61,7 +61,7 @@ fi
 }
 
 @test "import ntp puppet class" {
-  id=$(hammer --csv proxy list | tail -n1 | cut -d, -f1)
+  id=$(hammer --csv proxy list | grep $HOSTNAME | cut -d, -f1)
```
[the remark this answers is not in the document]
Comment (ehelms) [reply bats/fb-test-puppet.bats]: Oh yea, that makes sense.

File: bats/fb-test-puppet.bats [hunk 2]
```diff
@@ -61,7 +61,7 @@ fi
 }
 
 @test "import ntp puppet class" {
-  id=$(hammer --csv proxy list | tail -n1 | cut -d, -f1)
+  id=$(hammer --csv proxy list --name $HOSTNAME | cut -d, -f1)
```
Comment (ekohl) [inline bats/fb-test-puppet.bats]: Everywhere else we use `$(hostname -f)`. Is `$HOSTNAME` also defined and should we rely on that instead?

Comment (zjhuntin) [reply bats/fb-test-puppet.bats]: Sure, I think that makes the most sense.

Comment (ekohl) [reply bats/fb-test-puppet.bats]: I'd don't care which one we use, but I'd like it to be consistent. Could you update it?

**Comment by zjhuntin** [comment]
@ehelms @ekohl updated, figured out it was --search, not name we needed, and now it's consistent using $(hostname -f)

File: bats/fb-test-puppet.bats [hunk 3]
```diff
@@ -61,7 +61,7 @@ fi
 }
 
 @test "import ntp puppet class" {
-  id=$(hammer --csv proxy list | tail -n1 | cut -d, -f1)
+  id=$(hammer --csv proxy list --search $(hostname -f) | tail -n1 | cut -d, -f1)
```
Comment (ekohl) [inline bats/fb-test-puppet.bats]: I'd expect you should be able to use `hamer proxy info --name $(hostname -f)` and not need to use tail. You might even be able to simplify it all to `hammer proxy import-classes --name $(hostname -f)`.

Comment (zjhuntin) [reply bats/fb-test-puppet.bats]: Totally, right that works great.

**Pull request merged**